examples/openvino/stable-diffusion/requirements.txt

```diff
@@ -2,4 +2,4 @@ accelerate
 diffusers
 torch~=1.13
```

Comment:
Is there anything blocking for pytorch 2.0?

Reply:
Did not check it.